std/file.d

```diff
@@ -3902,6 +3903,15 @@ string tempDir() @trusted
             DWORD len = GetTempPathW(buf.length, buf.ptr);
             if (len) cache = toUTF8(buf[0 .. len]);
         }
+        else version(Android)
+        {
+            // Don't check for a global temporary directory as
+            // Android doesn't have one.  Instead, use the local
+            // directory or set cache here to your app's local
+            // directory.
+
+            cache = getcwd();
```

Comment:
You're right, I'll change it.  I was primarily building test runner apps, which need the hard-coded `tempDir` for test functions like `deleteme`, so I wasn't thinking straight about the general need for this, which doesn't exist.